install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/tasks/build_securedrop_app_code_deb.yml

```diff
@@ -1,12 +1,19 @@
 ---
+- name: Copy entire SecureDrop repo to build path.
+  synchronize:
+    src: "{{ role_path }}/../../../../"
+    dest: "{{ securedrop_app_code_deb_dir }}"
```

Comment:
We should only have the securedrop-1.0.0.tar.gz copied into the right place.

Reply:
Doesn't `dh-virtualenv` need the source tree, for `setup.py` at least?

Reply:
No, we need a release tarball (made using `setup.py`) to build debian packages.

Reply:
I still don't see how the sdist tarball is required, but the latest change uses it to create the directory where the package is built.

Reply:
We will have to create proper Debian packages following standard guidelines which is reproducible. Having a released source tar ball is the first step there.

Reply:
OK. It sounds to me like extracting that tarball from this process and publishing it is going to require work that I'd consider beyond the scope of this issue and PR. Could you file a separate issue with more details on the requirements for reproducibility?

Reply:
Yes, for this PR we will just have to fetch the tarball we are creating into the build directory. Making a comment in next few minutes.